types/helper_test.go

```diff
@@ -42,13 +39,14 @@ func (s *testTypeHelperSuite) TestStrToInt(c *C) {
 	}
 	for _, tt := range tests {
 		output, err := strToInt(tt.input)
```

Comment:
please make sure range and t.Parallel works right.
see https://github.com/pingcap/tidb/pull/27939 for deail